Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -435,14 +441,17 @@ public bool TryUpdateQueryBufferForLayerMask(LayerMask layerMask, Vector3 pointe
                         // Check to ensure the object is beyond the minimum distance
                         bool pastMinDistance = relativeColliderPosition.sqrMagnitude >= queryMinDistance * queryMinDistance;
 
-                        if (!pastMinDistance || !inAngle)
+                        bool isBoundHandler = ignoreHandlersForQuery && grabbable != null && grabbable.GetComponent<BoundsHandleVisual>() != null;
+
+                        if (!pastMinDistance || !inAngle || isBoundHandler)
                         {
                             grabbable = null;
                             continue;
                         }
                         
                         if (grabbable != null)
                         {
+                            Debug.Log(grabbable);
```

Comment:
Do we always want to log this grabbable, or was this mainly used for adhoc validation/verification?

Reply:
it was mostly for debug woops